bs-react-native-next/src/apis/NativeModules.bs.js

```diff
@@ -1,4 +1,4 @@
-// Generated by BUCKLESCRIPT VERSION 5.0.0, PLEASE EDIT WITH CARE
+
```

Comment:
why is there new line in some cases & some cases not?

Reply:
Different in bs-platform 5.0.1 for some reason unknown to me.